.cirrus.yml

```diff
@@ -49,7 +49,7 @@ except_nightly_cron: &EXCEPT_ON_NIGHTLY_CRON
 #
 build_task:
   eks_container:
-    <<: *CONTAINER_DEFINITION
+    <<: *CONTAINER_DEFINITION_17
```

Comment:
Would maybe be interesting to keep some QA axis on Java 11, at least until we officially move all our products to Java 17 as a baseline